src/test/resources/data/real/18EU_A.report

```diff
@@ -82,109 +82,109 @@ BuysItemFor,Martin,15,105
 Floats,15
  
 SelectForAuctioning,Volker,8
-PASSES,Martin
-PASSES,Arne
-PASSES,Jupp
+DeclinedToBid,Martin
+DeclinedToBid,Arne
+DeclinedToBid,Jupp
 ITEM_PRICE_REDUCED,8,90
-PASSES,Volker
-PASSES,Martin
+DeclinedToBid,Volker
+DeclinedToBid,Martin
 BuysItemFor,Arne,8,90
 Floats,8
  
 SelectForAuctioning,Martin,10
-PASSES,Arne
-PASSES,Jupp
-PASSES,Volker
+DeclinedToBid,Arne
+DeclinedToBid,Jupp
+DeclinedToBid,Volker
```

Comment:
Two things:
1.  Isn't this more than a message change, but a change to the logic?
2.  I think the value should be `DECLINED_TO_BID`

Reply:
Well, I wouldn't consider this a change to the logic except in the fact that it moves the `ReportBuffer.add()` call to under the switch for the two action types. It doesn't actually affect any game play logic

Reply:
There is a bit of inconsistency in the value name, ie `DECLINED_TO_BID` vs `DeclinedToBid`-- we have both variants in the code base so I picked the one was similar to the next closest, ie `DeclineToBid`. Would be nice to have this standardized but we have quite a bit a variance and I'd rather spend my time making substantial changes :)